src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -206,12 +207,14 @@ public async Task<OrchestratorResponse<ApprenticeshipDetailsViewModel>> GetAppre
 
                 apprenticeship.HashedAccountId = hashedAccountId;
 
+                var includeFrameworks = commitmentData.Commitment.ApprenticeshipEmployerTypeOnApproval != ApprenticeshipEmployerType.NonLevy;
+
                 return new OrchestratorResponse<ExtendedApprenticeshipViewModel>
                 {
                     Data = new ExtendedApprenticeshipViewModel
                     {
                         Apprenticeship = apprenticeship,
-                        ApprenticeshipProgrammes = await GetTrainingProgrammes(true)
+                        ApprenticeshipProgrammes = await GetTrainingProgrammes(includeFrameworks)
```

Comment:
do any tests need updating for this change. again it's V1 so there may be none

Reply:
No coverage